core/src/main/java/com/linecorp/armeria/server/VirtualHost.java

```diff
@@ -501,4 +501,12 @@ private String toString(boolean withTypeName) {
     String toStringWithoutTypeName() {
         return toString(false);
     }
+
+
+    ServerErrorHandler errorHandler() {
+        if (serverConfig == null) {
```

Comment:
How about passing `errorHandler` as a parameter when we create a `VirtualHost` so that we don't need to check if the `serverConfig` is null?

Reply:
Is that mean to add `ServerErrorHandler serverErorrHandler` as a property of VirtualHost?